bublik/data/models/config.py

```diff
@@ -87,17 +87,34 @@ def get_active_version(cls, config_type, config_name):
             is_active=True,
         ).first()
 
+    @classmethod
+    def get_all_versions(cls, config_type, config_name):
+        return (
+            Config.objects.filter(
+                type=config_type,
+                name=config_name,
+            )
+            .order_by('-is_active', '-created')
+            .values('id', 'version', 'is_active', 'description', 'created')
+        )
+
+    def activate(self):
+        active = self.__class__.get_active_version(self.type, self.name)
+        if active:
+            active.is_active = False
```

Comment:
This code in "activate" is a bit confusing, right?

Effectively in case of active being present you're re-activating it.

Reply:
No. I find the active config, deactivate it, and activate the current one instead.

Reply:
No. I find the active config, deactivate it, and activate the current one instead.